crates/bdk/src/wallet/mod.rs

```diff
@@ -94,6 +94,91 @@ pub struct Wallet<D = ()> {
     secp: SecpCtx,
 }
 
+/// A structure to update [`KeychainTxOutIndex`], [`TxGraph`] and [`LocalChain`] atomically.
```

Comment:
We will need to change this to something more understandable. This is no longer the `bdk_chain` crate.

```rust
/// A structure to update [`Wallet`].
```

Reply:
I'm thinking we can have something like:
```
/// A structure to update [`Wallet`].

/// It updates [`bdk_chain::KeychainTxoutIndex`], [`bdk_chain::TxGraph`] and [`bdk_chain::LocalChain`] atomically.
```